rmw_opensplice_cpp/src/types.cpp

```diff
@@ -392,6 +392,25 @@ rmw_ret_t OpenSpliceStaticPublisherInfo::get_status(
           offered_deadline_missed.total_count_change;
         break;
       }
+    case DDS::OFFERED_INCOMPATIBLE_QOS_STATUS:
+      {
+        DDS::OfferedIncompatibleQosStatus offered_incompatible_qos;
+        DDS::ReturnCode_t dds_return_code =
+          topic_writer->get_offered_incompatible_qos_status(offered_incompatible_qos);
+        rmw_ret_t from_dds = check_dds_ret_code(dds_return_code);
```

Comment:
`check_dds_ret_code` is an existing function that converts `DDS::ReturnCode_t` to `rmw_ret_t`; don't know why it was named such.